mica-search/src/main/java/org/obiba/mica/search/rest/EsQueryResultParser.java

```diff
@@ -75,15 +75,15 @@ public static EsQueryResultParser newParser(AggregationMetaDataResolver aggregat
               StatsAggregationResultDto.newBuilder() //
                 .setDefault(MicaSearch.StatsAggregationResultDataDto.newBuilder() //
                   .setCount(defaultStats.getCount()) //
-                  .setMin(defaultStats.getMin()) //
-                  .setMax(defaultStats.getMax()) //
-                  .setAvg(defaultStats.getAvg()) //
+                  .setMin(!Double.isInfinite(defaultStats.getMin()) ? defaultStats.getMin() : -1) //
+                  .setMax(!Double.isInfinite(defaultStats.getMax()) ? defaultStats.getMax() : -1) //
+                  .setAvg(!Double.isNaN(defaultStats.getAvg()) ? defaultStats.getAvg() : -1) //
                   .setSum(defaultStats.getSum()).build()) //
```

Comment:
min, max, avg are optional fields, so why not just ignoring them if infinite/nan is encountered?